### PR TITLE
Five new APIs to facilitate docbot Development

### PR DESCRIPTION
**Pull request by radkoff**
Document GET, Point GET/POST, and DocbotRecords GET/POST

Testing done: new Appwrite functions were set up in staging, and I was able to hit each one with curl. Confirmed that new Points/DocbotRecords are in the staging DB

OpenAPI docs aren't working right yet but I still added yaml files for the new APIs

### CODE REVIEW

File: appwrite.json [hunk 1]
```diff
@@ -184,6 +184,81 @@
...
+            "events": [],
+            "schedule": "",
+            "timeout": 60,
+            "commands": "npm install && npm run build"
+        },
+        {
+            "$id": "",
+            "name": "GET /point/v1",
+            "runtime": "node-16.0",
+            "path": "functions/internal/Point/GET/v1",
```
Comment (JustinBack) [inline appwrite.json:207]: Move from internal to public

File: functions/internal/Point/GET/v1/README.md
Comment (JustinBack) [inline functions/internal/Point/GET/v1/README.md:1]: Move to public

File: functions/internal/Point/GET/v1/gruntfile.js
Comment (JustinBack) [inline functions/internal/Point/GET/v1/gruntfile.js:1]: Move to public

File: functions/internal/Point/GET/v1/package-lock.json
Comment (JustinBack) [inline functions/internal/Point/GET/v1/package-lock.json:1]: Move to public

File: functions/internal/Point/GET/v1/package.json
Comment (JustinBack) [inline functions/internal/Point/GET/v1/package.json:1]: Move to public

File: functions/internal/Point/GET/v1/src/helpers/Phoenix.ts
Comment (JustinBack) [inline functions/internal/Point/GET/v1/src/helpers/Phoenix.ts:1]: Move to public

File: functions/internal/Point/GET/v1/src/index.ts
Comment (JustinBack) [inline functions/internal/Point/GET/v1/src/index.ts:1]: Move to public

File: functions/internal/Point/GET/v1/tsconfig.json
Comment (JustinBack) [inline functions/internal/Point/GET/v1/tsconfig.json:1]: Move to public

File: functions/internal/Point/spec/point.yaml
Comment (JustinBack) [inline functions/internal/Point/spec/point.yaml:1]: Move GET to public

**Review by JustinBack — CHANGES_REQUESTED** [review]

**Comment by radkoff** [comment]
Fixed a bug that was preventing POST content from being read in the bridge. Testing done: ran the bridge locally via Docker, using env variables to point to prod appwrite. Tested both new POSTs, and a GET to make sure that didn't break

**Pull request merged**